components/02-elements/cookie-message/cookie-message.js

```diff
@@ -1,46 +1,33 @@
-// TO DO - Needs a refactor
-
-'use strict'
-
-const messageClose = document.querySelector('.cookie-message__close'),
-      messageTop = document.querySelector('.cookie-message--top'),
-      cookieMessage = document.querySelector('.cookie-message--cookie');
-
-function closeBar() {
-  const cookieMessage = this.parentNode.parentNode;
-
-  if (cookieMessage.classList.contains('cookie-message--top')) {
-    localStorage.setItem('cookie-message-top', 1);
-    messageTop.style.height = 0;
-    messageTop.classList.add('closed');
+'use strict';
+class CookieMessage {
+  constructor() {
+    this.messageClose  = document.querySelector('.cookie-message__close');
+    this.cookies       = document.querySelectorAll('.cookie-message');
+
+    // todo domyslnie wylaczony, jezeli local storage nie ma wpisu to wlaczamy
```

Comment:
keep english comments

Reply:
oopsie , that should go away :D. Removed